src/deployments/utils/Logger.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: UNLICENSED
+// Copyright (c) 2025 Aave Labs
+pragma solidity ^0.8.0;
+
+import 'forge-std/StdJson.sol';
+import 'forge-std/Vm.sol';
+import {console} from 'forge-std/console.sol';
+import {DeployUtils} from 'src/deployments/utils/DeployUtils.sol';
+
+contract Logger is DeployUtils {
+  using stdJson for string;
+
+  struct AddressEntry {
+    string label;
+    address value;
+  }
+
+  struct ValueEntry {
+    string label;
+    uint256 value;
+  }
+
+  Vm private constant vm = Vm(address(bytes20(uint160(uint256(keccak256('hevm cheat code'))))));
```

Comment:
we define this here, so we don't need DeployUtils anymore. I would remove DeployUtils, wdyt?